src/Features/CSharp/Portable/ConvertToRawString/ConvertToRawStringHelpers.cs

```diff
@@ -75,10 +75,28 @@ public static bool IsInDirective(SyntaxNode? node)
         return false;
     }
 
-    public static bool CanConvert(VirtualCharSequence characters)
-        => !characters.IsDefault && characters.All(static ch => CanConvert(ch));
+    public static bool CanConvert(VirtualCharSequence characters, out bool containsEscapedEndOfLineCharacter)
```

Comment:
>containsEscapedEndOfLineCharacter [](http://example.com/codeflow?start=75&length=33)

It's ok for containsEscapedEndOfLineCharacter to have the wrong value if this method returns false?

Reply:
Yes. If we can't convert, we bail out. So all that matters is the value when this returns true.